src/components/security/security-controller.ts

```diff
@@ -1,14 +1,13 @@
 import { Request, Response } from "express";
-import { redactPhoneNumber } from "../../utils/strings";
 import { getManageGovukEmailsUrl } from "../../config";
 
-export function manageYourAccountGet(req: Request, res: Response): void {
+export function securityGet(req: Request, res: Response): void {
   const data = {
     email: req.session.user.email,
-    phoneNumber: redactPhoneNumber(req.session.user.phoneNumber),
+    phoneNumber: req.session.user.phoneNumber.slice(-4),
```

Comment:
Oh cool, nice one.
So does anything still use that `redactPhoneNumber` function in utils, or can we remove it?

Reply:
It's still referenced in a few controllers unfortunately! As we are still displaying the phone number in that format in a few places